app/models/chargeback_container_project.rb

```diff
@@ -62,7 +62,7 @@ def self.get_keys_and_extra_fields(perf, ts_key)
     project = @data_index.fetch_path(:container_project, :by_group_id, perf.resource_id)
     key = "#{project.id}_#{ts_key}"
 
-    [key, {"project_name"  => project.name}]
+    [key, {"project_name"  => project.old_ems_id.nil? ? project.name : "#{project.name} (Deleted)"}]
```

Comment:
Also, I wonder if `Archived` is a better term than `Deleted`, since that's the term we use elsewhere in the product.  @dclarizio Thoughts here?